meta-iot2000-bsp/conf/layer.conf

```diff
@@ -11,3 +11,5 @@ BBFILE_PRIORITY_iot2000-bsp = "6"
 
 IOT2000_MIT_LICENSE = "${LAYERDIR}/COPYING.MIT"
 IOT2000_GPLv2_LICENSE = "${LAYERDIR}/COPYING.GPLv2"
+
+PACKAGE_CLASSES = "package_ipk"
```

Comment:
b55834a3a0 suggest to do this in the local.conf - which is now the official way?